[RHSSO-1981] Update  RH-SSO server version to 7.5.3.GA & EAP server version to 7.4.6.GA

iankko opened the pull request:
[RHSSO-1981] Update:
    
    * RH-SSO server version to 7.5.3.GA,
    * EAP server version to 7.4.6.GA,
    * Wildfly Galleon plugin & Wildfly Galleon provisioning parent
      version to 5.2.6,
    * Templates version to 7.5.3.GA
    
    Also sync various labels and env vars to the values as present in the
    EAP 7.4 with OpenJDK 11 for OpenShift container image
    
    Signed-off-by: Jan Lieskovsky <jlieskov@redhat.com>

Testing: A candidate RH-SSO container image containing this change was built within \#587 container pipeline run.


Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

Commit:
[RHSSO-1981] Update:

* RH-SSO server version to 7.5.3.GA,
* EAP server version to 7.4.6.GA,
* Wildfly Galleon plugin & Wildfly Galleon provisioning parent
  version to 5.2.6,
* Templates version to 7.5.3.GA

Also sync various labels and env vars to the values as present in the
EAP 7.4 with OpenJDK 11 for OpenShift container image

Signed-off-by: Jan Lieskovsky <jlieskov@redhat.com>

ASzc reviewed (approved):
Seems ok. There were a few things that seemed weird, but upon checking them out, they're fine

iankko commented:
Scratch/candidate OSBS build of RH-SSO container image with this change is available in container pipeline \#588 run.

iankko commented:
Updated to RH-SSO 7.5.3.CR2 and also updated tag of `jboss-eap-modules.git` to most recent `EAP_746_CR2_3` and tag of `jboss-eap-7-image.git` to the moast recent `EAP_LP_746_CR1`.

Merging based on the preliminary RH-SSO candidate image testing, which looks good.
